guides/v3.4.0/components/the-component-lifecycle.md

```diff
@@ -246,7 +247,7 @@ export default Component.extend({
 
   didRender() {
     this._super(...arguments);
-    this.$('.item-list').scrollTop(this.$('.selected-item').position().top);
+    this.element.querySelector('.item-list').scrollTop = this.element.querySelector('.selected-item').offsetTop);
```

Comment:
I'm not sure this works, as offsetTop will determine use the document or nearest *positioned* document. Not sure you can assume that the .item-list would be positioned. Here is how I tackled this:
```
const scrollTarget = Math.abs(this.element.getBoundingClientRect().top - this.element.querySelector('.selected-item').getBoundingClientRect().top);
this.element.querySelector('.item-list').scrollTop = scrollTarget;   
```